tests/FacebookClientTest.php

```diff
@@ -72,21 +72,17 @@ public function testACustomHttpClientCanBeInjected()
     {
         $handler = new MyFooClientHandler();
         $client = new FacebookClient($handler);
-        $httpHandler = $client->getHttpClientHandler();
+        $httpHandler = $client->getHttpClient();
```

Comment:
I'd rename the variable to `$httpClient` for consistency.
BTW, the "handler" naming doesn't make sense anymore, so I'd rename `MyFooClientHandler` to `MyFooHttpClient` and rename `$handler` to `$customHttpClient`, or remove it has it's used only once :)